test/e2e/app_management_test.go

```diff
@@ -141,6 +141,25 @@ func TestAppCreation(t *testing.T) {
 		})
 }
 
+func TestDeleteAppResource(t *testing.T) {
+	ctx := Given(t)
+
+	ctx.
+		Path(guestbookPath).
+		When().
+		Create().
```

Comment:
I think the test is failing because `.Sync()` is missing.

Reply:
Thanks. Updated.